### ISSUE TITLE
shift-tab does not show function docs

### ISSUE THREAD

**Issue by mmagnuski** [spyder-ide/spyder-notebook#315]
## Description of your problem

**What steps will reproduce the problem?**

1. open Spyder, create empty notebook
2. evaluate `import numpy as np` in the first cell
3. type `np.arange(` and then press `shift + tab`

**What is the expected output? What do you see instead?**

I expect a small window to open with function documentation, but nothing happens.
This functionality worked in the previous jupyter-notebook release (`0.2.3`) - but I can't seem to get this previous release working with Spyder 4.

## Versions and main components

* Spyder Version:  4.1.5
* Python Version: 3.7.3
* Qt Version: 5.9.7
* PyQt Version: 5.9.2 (pyqt5-sip=12.8.0)
* Operating system: Windows

### jupyter:
```
# Name                    Version                   Build  Channel
jupyter                   1.0.0                    py37_7
jupyter_client            6.1.6                      py_0
jupyter_console           6.2.0                      py_0
jupyter_core              4.6.3                    py37_0
jupyterlab                2.2.6                      py_0
jupyterlab_pygments       0.1.1                      py_0
jupyterlab_server         1.2.0                      py_0
notebook                  6.1.1                    py37_0
spyder-notebook           0.3.0                      py_0    spyder-ide
```

## Dependencies

```
# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.6.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  6.0.6 (OK)
numpydoc >=0.6.0               :  0.9.1 (OK)
paramiko >=2.4.0               :  2.7.2 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.6.3 (OK)
pygments >=2.0                 :  2.7.1 (OK)
pylint >=1.0                   :  2.6.0 (OK)
pyls >=0.34.0;<1.0.0           :  0.35.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.7 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.2.1 (OK)
spyder_kernels >=1.9.4;<1.10.0 :  1.9.4 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.2 (OK)

# Optional:
cython >=0.21                  :  0.29.12 (OK)
matplotlib >=2.0.0             :  3.1.0 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  0.25.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  None (OK)

# Spyder plugins:
spyder_notebook                :  0.3.0 (OK)
```

**Comment by jitseniesen** [comment]
Thanks for the report. I can reproduce this issue, I will have a look at it. I'm hopeful that it's easy to fix.

**Comment by jitseniesen** [comment]
This is proving more complicated than I thought. The Shift-Tab functionality is provided by the tooltip module in Jupyter Lab, which is not yet integrated with the plugin. I want to release the next version soon to fix #314 so I am postponing this to the version after that.

**Comment by mmagnuski** [comment]
:tada:
